components/cam/src/physics/rrtmg/radiation.F90

```diff
@@ -1735,49 +1731,47 @@ subroutine radiation_tend( state, ptend,pbuf, &
                   endif
                 enddo
               enddo
-              if (use_SPCAM) then 
-                do m=1,crm_nz
-                   k = pver-m+1
-                   do i=1,ncol
-                      cld_tau_crm(i,ii,jj,m) =  cld_tau(rrtmg_sw_cloudsim_band,i,k)
-                   end do ! i
-                end do ! m
-              endif 
             else  ! cldfsnow_idx > 0
               c_cld_tau    (1:nbndsw,1:ncol,:) = cld_tau    (:,1:ncol,:)
               c_cld_tau_w  (1:nbndsw,1:ncol,:) = cld_tau_w  (:,1:ncol,:)
               c_cld_tau_w_g(1:nbndsw,1:ncol,:) = cld_tau_w_g(:,1:ncol,:)
               c_cld_tau_w_f(1:nbndsw,1:ncol,:) = cld_tau_w_f(:,1:ncol,:)
             endif  ! cldfsnow_idx > 0
 
+            ! Save cloud optical depth for CRM column to output later
```

Comment:
This was previously inside the check on whether or not snow fraction is present, but we want to save the CRM optical properties regardless of whether or not snow is present.

Reply:
ah, that makes sense, thanks